### PR TITLE
[PEER-608] Add search similar questions when creating the new one

### PR DESCRIPTION
**Pull request by vits89**

### CODE REVIEW

**Review by GoodPassiveMan — APPROVED** [review]

File: app/components/QuestionForm/ExistingQuestions.jsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+  &:last-child {
+    border: none;
+  }
+`;
+
+const ExistingQuestions = ({ questions, skip, intl }) => (
+  <Container>
+    <Label>
+      {intl.formatMessage(messages.existingQuestionsLabel)}{' '}
```
[the remark this answers is not in the document]
Comment (vits89) [reply app/components/QuestionForm/ExistingQuestions.jsx:36]: Yes, this is OK. But I can use margin-right property. Should I use it?

[the remark this answers is not in the document]
Comment (GoodPassiveMan) [reply app/components/QuestionForm/ExistingQuestions.jsx:36]: I approved this PR, so at your discretion.

[the remark this answers is not in the document]
Comment (vits89) [reply app/components/QuestionForm/ExistingQuestions.jsx:36]: OK.

**Review by sergeyi23 — COMMENTED** [review]
Looks good to me.

**Pull request merged**